golang/cosmos/daemon/cmd/testnet.go

```diff
@@ -48,13 +48,13 @@ func testnetCmd(mbm module.BasicManager, genBalIterator banktypes.GenesisBalance
 	cmd := &cobra.Command{
 		Use:   "testnet",
 		Short: fmt.Sprintf("Initialize files for a %s testnet", AppName),
-		Long: `testnet will create "v" number of directories and populate each with
+		Long: `testnet will create one directory per validator and populate each with
 necessary files (private validator, genesis, config, etc.).
 
 Note, strict routability for addresses is turned off in the config file.
 
 Example:
-	agd testnet --v 4 --output-dir ./output --starting-ip-address 192.168.10.2
+	agd testnet -v 4 --output-dir ./output --starting-ip-address 192.168.10.2
```

Comment:
Please don't use `-v`... that's a conventional alias for `--verbose`.  Maybe something like `-V` would be okay, or even better `-n`?